app/src/views/auth/Signin.vue

```diff
@@ -39,15 +38,6 @@ export default class Signin extends Mixins(LoadKeycloakRolesMixin, NrAffiliation
     this.loadKeycloakRoles()
     await this.updateUser()
 
-    // if there is stored NR data to process then affiliate it now
-    const nr = JSON.parse(sessionStorage.getItem('NR_DATA'))
-    if (nr) {
-      // Use the new "magic link routes" in the BRD to perform the affiliations and draft creations.
-      Navigate(this.magicLink(nr))
-      // clear NR data for next time
-      sessionStorage.removeItem('NR_DATA')
-    }
-
```

Comment:
Please test this by:
1. make sure you are not logged in
2. create a NR
3. in a different window (as staff), examine and approve the NR
4. in the original window, reload the NR (see example below)
5. click Register Your Business
6. log in
7. when you are returned to Namerequest UI, is the Nr handled automatically? (it might be this signin code, or it might be the created/mounted code)

<img width="749" height="295" alt="image" src="https://github.com/user-attachments/assets/a7b5f3f6-3d9e-4184-9b97-f72a1b8b993e" />

<img width="1035" height="551" alt="image" src="https://github.com/user-attachments/assets/15e03d49-3dac-4002-9edf-1189e4621f4d" />

Reply:
I just tried this (in Dev) and it failed, but the task trace shows that it came through created (not signin), so this may be OK.

Can you think of any other path that would hit signin before (or instead of) created?

Reply:
I'd like Bobby to test this but I think it's resolved. Let me know what you find.